tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot

```diff
@@ -0,0 +1,24 @@
+*** Settings ***
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Test Cases ***
+Verify "logged in users" are displayed in the dashboard
+  [Tags]  Sanity    ODS-354
+  Launch JupyterHub From RHODS Dashboard Dropdown
```

Comment:
I think it should be replaced by "Can Launch Jupyterhub" to address the change in Dashboard links to launch applications.